ml_git/commands/wizard.py

```diff
@@ -44,7 +44,7 @@ def wizard_for_field(context, field, input_message, required=False, wizard_flag=
     else:
         try:
             new_field = check_empty_for_none(request_new_value(input_message, required, type))
-            if not new_field:
+            if not new_field or new_field.isspace():
                 return default
             return new_field
```

Comment:
To avoid possible problems, we can trim the value informed before returning it.

Reply:
About the tests, I will keep that in mind, and about the trim, do you mean removing all external whitespaces from the string and modify to comparate with an empty string in line 47?

Reply:
No, just remove the outer spaces the user has passed in the field. In line 49.

```suggestion
            return new_field.strip()
```

Reply:
Changes have been made